splunk-otel-android/src/main/java/com/splunk/rum/PostApi29NetworkDetector.java

```diff
@@ -30,18 +30,21 @@
 import androidx.annotation.RequiresApi;
 import androidx.core.app.ActivityCompat;
 
-@RequiresApi(api = Build.VERSION_CODES.Q)
+@RequiresApi(api = Build.VERSION_CODES.P)
 class PostApi29NetworkDetector implements NetworkDetector {
```

Comment:
Should we rename the class to `PostApi28...`?

Reply:
Absolutely. Good call.